src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -386,8 +392,9 @@ class TooltipHTMLContent {
     }
 
     hide() {
-        this.el.style.display = 'none';
+        this.el.style.opacity = '0';
         this._show = false;
+        this._longHideTimeout = setTimeout(() => this._longHide = true, 500) as any;
     }
```

Comment:
I think we still need to set `display: 'none'` when `this._longHide = true`. I'm not sure if it will cause other issues like cover the graph and cause interaction issues. if only set the opacity to `0`.

Also, should the time `500ms` plus user-set hideDelay?

Reply:
> I think we still need to set `display: 'none'` when `this._longHide = true`. I'm not sure if it will cause other issues like cover the graph and cause interaction issues. if only set the opacity to `0`.

Perhaps, you are worried that it will cause interaction issues if only set `opacity` to `0` for `opacity: 0` will still respond to DOM events. I suppose this issue could be solved through setting `visibility` to `hidden`. It won't respond to DOM events. So it should not bring interaction issues anymore. See this commit f038591383293467900c791865f4cb284c315acf

> Also, should the time `500ms` plus user set `hideDelay`?

This logic is now in the `hide` function, `hide` is called by `hideLater`, and `hideLater` has applied the `hideDelay`.
So I think no need to plus the specified `hideDelay` here again.

Reply:
I think it's a solution I didn't think of. Looks good! Nice work!